src/test/java/seedu/guest/logic/parser/AddCommandParserTest.java

```diff
@@ -48,38 +52,38 @@ public void parse_allFieldsPresent_success() {
 
         // whitespace only preamble
         assertParseSuccess(parser, PREAMBLE_WHITESPACE + NAME_DESC_BOB + PHONE_DESC_BOB + EMAIL_DESC_BOB
-                + DATE_RANGE_DESC_BOB + NUMBER_OF_GUESTS_DESC_BOB, new AddCommand(expectedGuest));
+                + DATE_RANGE_DESC_BOB + NUMBER_OF_GUESTS_DESC_BOB + IS_ROOM_CLEAN_DESC_BOB,
+                new AddCommand(expectedGuest));
 
         // multiple names - last name accepted
         assertParseSuccess(parser, NAME_DESC_AMY + NAME_DESC_BOB + PHONE_DESC_BOB + EMAIL_DESC_BOB
-                + DATE_RANGE_DESC_BOB + NUMBER_OF_GUESTS_DESC_BOB, new AddCommand(expectedGuest));
+                + DATE_RANGE_DESC_BOB + NUMBER_OF_GUESTS_DESC_BOB + IS_ROOM_CLEAN_DESC_BOB,
+                new AddCommand(expectedGuest));
 
         // multiple phones - last phone accepted
         assertParseSuccess(parser, NAME_DESC_BOB + PHONE_DESC_AMY + PHONE_DESC_BOB + EMAIL_DESC_BOB
-                + DATE_RANGE_DESC_BOB + NUMBER_OF_GUESTS_DESC_BOB, new AddCommand(expectedGuest));
+                + DATE_RANGE_DESC_BOB + NUMBER_OF_GUESTS_DESC_BOB + IS_ROOM_CLEAN_DESC_BOB,
+                new AddCommand(expectedGuest));
 
         // multiple emails - last email accepted
         assertParseSuccess(parser, NAME_DESC_BOB + PHONE_DESC_BOB + EMAIL_DESC_AMY + EMAIL_DESC_BOB
-                + DATE_RANGE_DESC_BOB + NUMBER_OF_GUESTS_DESC_BOB, new AddCommand(expectedGuest));
+                + DATE_RANGE_DESC_BOB + NUMBER_OF_GUESTS_DESC_BOB + IS_ROOM_CLEAN_DESC_BOB,
+                new AddCommand(expectedGuest));
 
         // multiple date ranges - last date range accepted
         assertParseSuccess(parser, NAME_DESC_BOB + PHONE_DESC_BOB + EMAIL_DESC_BOB + DATE_RANGE_DESC_AMY
-                        + DATE_RANGE_DESC_BOB + NUMBER_OF_GUESTS_DESC_BOB,
+                + DATE_RANGE_DESC_BOB + NUMBER_OF_GUESTS_DESC_BOB + IS_ROOM_CLEAN_DESC_BOB,
                 new AddCommand(expectedGuest));
 
         // multiple numbers of guests - last number of guests accepted
         assertParseSuccess(parser, NAME_DESC_BOB + PHONE_DESC_BOB + EMAIL_DESC_BOB
                 + DATE_RANGE_DESC_BOB + NUMBER_OF_GUESTS_DESC_AMY
-                + NUMBER_OF_GUESTS_DESC_BOB, new AddCommand(expectedGuest));
-    }
+                + NUMBER_OF_GUESTS_DESC_BOB + IS_ROOM_CLEAN_DESC_BOB, new AddCommand(expectedGuest));
 
-    @Test
-    public void parse_optionalFieldsMissing_success() {
```

Comment:
Why was this test deleted?

Reply:
We no longer have optional fields as such this test is redundant. I have checked with Zong Xian (who removed the Tag field which was optional) and he agreed to remove it.